channeldb/payments_test.go

```diff
@@ -175,3 +177,261 @@ func TestRouteSerialization(t *testing.T) {
 			spew.Sdump(testRoute), spew.Sdump(route2))
 	}
 }
+
+// deletePayment removes a payment with paymentHash from the payments database.
+func deletePayment(db *DB, paymentHash lntypes.Hash) error {
```

Comment:
nit: pass `testing.T` into this and just `Fatal` directly if we have an error. Nice way of gatekeeping the function for testing only as well.

Reply:
Gatekeeping was exactly what I wanted to achieve, thank you for the hint. Also added `t.Helper()`.